Assets/MRTK/Core/Utilities/StandardShader/ClippingPrimitive.cs

```diff
@@ -16,10 +17,17 @@ namespace Microsoft.MixedReality.Toolkit.Utilities
     [HelpURL("https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Rendering/ClippingPrimitive.html")]
     public abstract class ClippingPrimitive : MonoBehaviour, IMaterialInstanceOwner
     {
+        private const int InitialCollectionSize = 256;
+
+        /// <summary>
+        /// The renderer(s) that should be affected by the primitive. This collection is a copy of internal HashSet.
```

Comment:
nit: little misleading since this is not a HashSet, it's a Dictionary.

Reply:
Indeed, it's my slip-up 😅 I didn't notice.